app/assets/stylesheets/darkswarm/_shop-product-thumb.scss

```diff
@@ -2,9 +2,16 @@
   products {
     product {
       .product-thumb {
-        width: 7rem;
-        height: 7rem;
+        // Desktop: the product summary is nine columns wide. Use two
+        // for the image. In our design, that's a max of 192px.
+        width: calc(100% / 9 * 2);
```

Comment:
I'd move the calculation to the comment and just have width: 22.2%

Reply:
I think this part is not relevant and could be removed:
"In our design, that's a max of 192px."